docs/content/admin/_index.en.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Administration"
+date: 2022-12-21T10:54:47-06:00
+lastmod: 2022-12-21T10:54:47-06:00
+description: "A guide for TRISA Administrators"
+weight: 100
+---
+
+The TRISA Global Directory Service is managed by administrators appointed by the TRISA board to support and vet VASPs registering to join the TRISA network. TRISA Admins have the following primary responsibilities:
+
+1. Review new VASP registrations and vet them for access.
+2. Administrate the certificate authority on behalf of TRISA.
+3. Provide technical support and advice to VASPs.
+4. Manage the GDS system and TestNet.
+
+This section of the documentation is specifically focused on the workflows for TRISA admins.
+
+## Admin UI
```

Comment:
Maybe one additional short paragraph to the effect of:

Access to the Admin UI is limited to TRISA administrators; it is not accessible to the general public or members of the TRISA network. If you wish to view the public-facing landing page for the TRISA Global Directory Service, please visit vaspdirectory.net.  If you require support accessing your TRISA administrator account to the Admin UI, please contact support@rotational.io.